### PR TITLE
Adding some admin

### PR DESCRIPTION
**Pull request by dchaplinsky**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -24,4 +24,4 @@ veryprettytable==0.8.1
 webassets==0.12.1
 Werkzeug==0.12.2
 wheel==0.29.0
-yuicompressor==2.4.8
+Flask-Admin==1.5.0
```
Comment (mrgambal) [inline requirements.txt]: You'd better put it in alphabet order.

File: vulyk/admin/models.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+from flask_admin.contrib.mongoengine import ModelView
```
Comment (mrgambal) [inline vulyk/admin/models.py]: Encoding and `__all__`.

File: vulyk/bootstrap/_admin.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import flask_login as login
```
Comment (mrgambal) [inline vulyk/bootstrap/_admin.py]: Encoding and `__all__`.
Split imports with space.

File: vulyk/admin/models.py [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+import flask_login as login
+
+
+class AuthModelView(ModelView):
+    def is_accessible(self):
```
Comment (mrgambal) [inline vulyk/admin/models.py]: Please, add type hints wherever  you added some methods/procedures.

File: vulyk/settings.py [hunk 5]
```diff
@@ -109,3 +109,5 @@
         "url_prefix": "gamification"
```
Comment (mrgambal) [inline vulyk/settings.py]: This one is from the previous commit, but still - fix double quotes here.

File: vulyk/bootstrap/__init__.py [hunk 6]
```diff
@@ -52,6 +52,12 @@ def init_app(name):
 
         _assets.init(app)
         _social_login.init_social_login(app, db)
+
+        if app.config.get('ENABLE_ADMIN', False):
+            from . import _admin
+            admin = _admin.init_admin(app)
```
Comment (mrgambal) [inline vulyk/bootstrap/__init__.py]: Why did you split declaration and assignment? Is there anything under the hood?

File: vulyk/blueprints/gamification/__init__.py [hunk 7]
```diff
@@ -35,6 +37,12 @@ def __init__(self, *args, **kwargs):
             k: 1 if k == 1 else (k - 1) * 25 for k in range(1, 51)
         }
 
+    def register(self, app, options, first_registration=False):
+        super().register(app, options, first_registration)
+
+        if app.config.get('ENABLE_ADMIN', False):
+            app.admin.add_view(AuthModelView(FundModel))
```
Comment (mrgambal) [inline vulyk/blueprints/gamification/__init__.py]: Could `app.admin` by any chance happen to be uninitialized here?
It'd be better to add some error handling in here with proper exception being wrapped and re-thrown with meaningful message.

Also, why don't you bind `RuleModel` to admin panel?

Comment (dchaplinsky) [inline vulyk/blueprints/gamification/__init__.py]: That's TBA.

File: vulyk/bootstrap/__init__.py (hunk 6 above)
Comment (dchaplinsky) [inline vulyk/bootstrap/__init__.py]: There was.

**Pull request closed without merging**